src/main/java/io/tiledb/spark/TileDBDataWriter.java

```diff
@@ -356,14 +379,18 @@ private boolean writeRecordToBuffer(
       case TILEDB_STRING_ASCII:
       case TILEDB_STRING_UTF8:
         {
-          String val = record.getString(ordinal);
+          String val = " "; // necessary if val is null
+          try {
```

Comment:
Can we eliminate this `try` clause here? In case of a `null` value, shouldn't `record.getString(ordinal)` just return a `null` value?

Reply:
When the record has a null value, record.getString() throws a NullPointerException it does not return null. I have tried to avoid the try{} but it was not possible.

Reply:
As we already check whether that specific value is `null` (`record.isNullAt(ordinal)`) let's keep track of it using a `boolean` flag (e.g. `isNull` or something similar). We can then replace the `try` clause with an `if` condition.

Reply:
Done